internal/ingress/controller/store/store.go

```diff
@@ -823,12 +823,6 @@ func (s *k8sStore) syncIngress(ing *networkingv1.Ingress) {
 	copyIng := &networkingv1.Ingress{}
 	ing.ObjectMeta.DeepCopyInto(&copyIng.ObjectMeta)
 
-	klog.Errorf("Blocklist: %v", s.backendConfig.AnnotationValueWordBlocklist)
-	if err := checkBadAnnotationValue(copyIng.Annotations, s.backendConfig.AnnotationValueWordBlocklist); err != nil {
```

Comment:
Hum, why you removed this? Those are different things :)

The Checkngress on the controller/controller.go is used by the admission webhook.

Here is used during the ingress reconciliation (once the object is already persisted), so it's required.

Would leave something like:

```golang
if s.backendConfig.AnnotationValueWordBlocklist != ""{ 
  if err := checkBadAnnotationValue(copyIng.Annotations, s.backendConfig.AnnotationValueWordBlocklist); err != nil {
		klog.Errorf("skipping ingress %s: %s", key, err)
		return
  }
}
```